ephios/settings.py

```diff
@@ -224,6 +227,8 @@
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 
 STATIC_URL = env.str("STATIC_URL", default="/static/")
+MEDIA_URL = env.str("MEDIA_URL", default="/usercontent/")
+FALLBACK_MEDIA_SERVING = env.bool("FALLBACK_MEDIA_SERVING", default=False)
```

Comment:
So people have a working dev setup I'd suggest (similar to how django handles static file serving)

```suggestion
FALLBACK_MEDIA_SERVING = env.bool("FALLBACK_MEDIA_SERVING", default=DEBUG)
```